pkg/storage/engine.go

```diff
@@ -17,6 +18,7 @@ type EngineType string
 // Engine represents a storage engine.
 type Engine interface {
 	v1.IssuerService
+	types.UserInfoService
```

Comment:
This was kind of a mistake on my part: By embedding all of these types in a single interface, we need to make the names of their methods clear. Right now `IssuerService` just has vague methods like `GetByURI`, which make sense in the context of an `IssuerService` but not an overall storage engine. In the interest of making that less bad, we can make sure all method names in `UserInfoService` contain `UserInfo`.